infra.go

```diff
@@ -119,19 +119,20 @@ func (i *InfrastructureResource) ResourceType() TaggableResource {
 }
 
 func (client *Client) CreateInfrastructure(input InfraInput) (*InfrastructureResource, error) {
+	data := JSON(input.Data)
```

Comment:
`NewJSON()` is already taken, https://github.com/OpsLevel/opslevel-go/blob/89e50eba5e929121d43f633bb10f0d9bf0b100e9/json.go#L15